app/javascript/react/utils/datesHelper.tsx

```diff
@@ -0,0 +1,24 @@
+import moment from "moment";
+
+const today = moment();
+
+const getFullWeeksOfMonth = (year: number, month: number): string[][] => {
```

Comment:
When using this I can see the calendar end with the current date (ex. today 2.02) and as the design shows, the considered month should end with the last day from considered month, no more right?

Reply:
As I answered to your comment from previous CR - I don't think it should end with the last day. I think it's just simplification of the design that all months start a at the very same day of the week, all of them are 31 day long and all of them ends at the very same day -  which happens to be at the at the bottom right corner. :) 
But I'm adding @franiakat to this discussion :)

Reply:
I'm with @smialko on that. Although worth checking with Iwona in the future.